packages/node_modules/@webex/react-component-activity-text/src/styles.css

```
@@ -53,3 +53,15 @@
 .activityText p {
   margin: 0;
 }
+
+.activityText p:not(:last-of-type) {
+  margin-bottom: 10px;
+}
```

```suggestion
.activityText p+p {
  margin-top: 10px;
}
```
Just a different way of doing it.

---

I just copied the web client's css code, but I'll look into that format, not familiar with it.